gnomad_qc/v4/subset.py

```diff
@@ -272,6 +275,10 @@ def main(args):
                     nhomalt=ht.subset_callstats_adj.homozygote_count[1:],
                 )
             )
+            ht = ht.checkpoint(
```

Comment:
Does this help with runtime? Just curious why it's written out.

Reply:
Not really, but I figured it would be nice to have as a table, and I also ended up needing to rerun the export and wanted to avoid needing to recompute callstats if I needed to do it again.